applications/fishing-map/src/features/resources/resources.slice.ts

```diff
@@ -6,25 +6,21 @@ import GFWAPI from '@globalfishingwatch/api-client'
 import { DataviewDatasetConfig, DatasetTypes } from '@globalfishingwatch/dataviews-client'
 
 export interface ResourceQuery {
+  id: string
```

Comment:
Why is that needed? IMHO we either use the URL as a unique id, or we use an id, but not both. 
Maybe I'm missing something but I'd rather call `resolveEndpoint` every time I need to get a URL to get a resource for a dataview, rather than having a double system.

Reply:
Good catch, I created a `resolveDataviewDatasetResource` to get the same url in different places of the app easily and removed the `id` see https://github.com/GlobalFishingWatch/frontend/pull/118/commits/de346d3f6e90567c076e9658922f83ced5e4aea5